doc/whats_new.rst

```diff
@@ -113,6 +113,8 @@ API
 
     - The ``compensation`` parameter in :func:`mne.io.read_raw_fif` has been deprecated in favor of the method :meth:`mne.io.Raw.apply_gradient_compensation` by `Eric Larson`_
 
+    - Added :class:`mne.decoding.Vectorizer` to convert the output of an MNE transformer into two dimensions so as to pipeline it to scikit-learn estimators by `Asish Panda`_
+
```

Comment:
@kaichogami please see here all unaddressed comments. Come on try to be reactive so we can iterate faster. These are 20 mins of your time.